AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3961,60 +3998,63 @@ function Debug-AzStorageAccountEntraKerbAuth {
             } catch {
                 $checks["CheckEntraObject"].Result = "Failed"
                 $checks["CheckEntraObject"].Issue = $_
-                Write-Error "CheckEntraObject - FAILED"
-                Write-Error $_
+                $aadTryCatchError = $_
+                Write-FailedPSStyle($aadTryCatchError)                
             }
         }
         #
         #Check if Reg key is enabled
         #
         if (!$filterIsPresent -or $Filter -match "CheckRegKey")
         {
+            [string] $aadObjIntro = "Checking Registry Key"
+            Write-Host $aadObjIntro
             try {
                 $checksExecuted += 1;
                 Write-Verbose "CheckRegKey - START"

```

Comment:
We could get rid of all the Write-Verbose with "- START" because they're just duplicating the functionality that we now have with your intro messages

Reply:
And same for "- END", if any